core/src/main/java/org/elasticsearch/search/fetch/subphase/highlight/AbstractHighlighterBuilder.java

```diff
@@ -119,10 +127,12 @@ protected AbstractHighlighterBuilder(StreamInput in) throws IOException {
         order(in.readOptionalWriteable(Order::readFromStream));
         highlightFilter(in.readOptionalBoolean());
         forceSource(in.readOptionalBoolean());
+        boundaryScannerType(in.readOptionalWriteable(BoundaryScannerType::readFromStream));
```

Comment:
Nice, I didn't see this pattern in the files I changed, so had no idea :). I will do that in all 4 places you commented on.